upcloud/request/managed_database.go

```diff
@@ -0,0 +1,549 @@
+func (m *ManagedDatabasePropertiesRequest) SetStringSlice(name string, value []string) *ManagedDatabasePropertiesRequest {
+	return m.Set(name, value)
+}
+
+// SetAutoUtilityIPFilter enables or disables automatic utility network ip filtering.
+// See upcloud.ManagedDatabasePropertyAutoUtilityIPFilter for more information.
+func (m *ManagedDatabasePropertiesRequest) SetAutoUtilityIPFilter(enabled bool) *ManagedDatabasePropertiesRequest {
+	return m.SetBool(upcloud.ManagedDatabasePropertyAutoUtilityIPFilter, enabled)
+}
+
+// SetIPFilter sets the list of allowed host or networks that can access the service.
+//
+// Use upcloud.ManagedDatabaseAllIPv4 to enable access from anywhere.
+//
+// See upcloud.ManagedDatabasePropertyIPFilter for more information.
+func (m *ManagedDatabasePropertiesRequest) SetIPFilter(addressOrNetworkWithCIDRMask ...string) *ManagedDatabasePropertiesRequest {
+	return m.SetStringSlice(upcloud.ManagedDatabasePropertyIPFilter, addressOrNetworkWithCIDRMask)
+}
+
+// SetPublicAccess enables or disables public access from the internet.
+// See upcloud.ManagedDatabasePropertyPublicAccess for more information.
+func (m *ManagedDatabasePropertiesRequest) SetPublicAccess(enabled bool) *ManagedDatabasePropertiesRequest {
+	return m.SetBool(upcloud.ManagedDatabasePropertyPublicAccess, enabled)
+}
+
+// Get returns a property value by name. The underlying map is initialised if it's nil
+func (m *ManagedDatabasePropertiesRequest) Get(name string) interface{} {
+	if m == nil {
+		return nil
+	}
+	if *m == nil {
+		*m = make(ManagedDatabasePropertiesRequest)
+	}
+	return (*m)[name]
+}
+
+// GetString returns a string property value. An empty string is returned for non-existing or not-a-string values.
+// The underlying map is initialised if it's nil.
+func (m *ManagedDatabasePropertiesRequest) GetString(name string) string {
+	v, _ := m.Get(name).(string)
+	return v
+}
+
+// GetInt returns an integer property value. A zero is returned for non-existing or not-an-integer values.
+// The underlying map is initialised if it's nil.
+func (m *ManagedDatabasePropertiesRequest) GetInt(name string) int {
+	v, _ := m.Get(name).(int)
+	return v
+}
+
+// GetBool returns a boolean property value. A false is returned for non-existing or not-a-boolean values.
+// The underlying map is initialised if it's nil.
+func (m *ManagedDatabasePropertiesRequest) GetBool(name string) bool {
+	v, _ := m.Get(name).(bool)
+	return v
+}
+
+// GetStringSlice returns a string-slice property value. A nil slice is returned for non-existing or not-a-string-slice values.
+// The underlying map is initialised if it's nil.
+func (m *ManagedDatabasePropertiesRequest) GetStringSlice(name string) []string {
+	if slice, ok := m.Get(name).([]string); ok {
+		return slice
+	}
+	// The slice might be a slice of empty interfaces if upcloud.ManagedDatabaseProperties is converted into
+	// request.ManagedDatabasePropertiesRequest
```

Comment:
These two containers seem very close to each other and duplicate some code, could we DRY them up or if they do have some differences making it impossible, making the differences clear?

Reply:
I'm a bit unfamiliar with the overall idea of this project's module layout. If you are ok with referencing upcloud pkg from request then yes. In fact, it seems I already do this.. :)

Reply:
Ahh, right, didn't notice that they are in different packages.. Up to you, in different packages I don't feel it's that crucial to DRY up.